### DISCUSSION TITLE
trinity error

### DISCUSSION THREAD

**Discussion by meganewing** [sr320/course-fish546-2021#118] [Q&A] [answered]
trying to run the perl line to get the trinity matrix but when it finishes running I can't seem to find the output anywhere, there is an error message that shows up in my jupyter notebook but it doesn't look like a normal input. 

<img width="1031" alt="Screen Shot 2021-03-04 at 6 12 45 PM" src="https://user-images.githubusercontent.com/66333757/110057301-3ad8bd00-7d15-11eb-9a31-2d6f1ee8af26.png">

**Comment by kubu4** [comment]
It needs an R library that it can't find. Not entirely sure how this works within Jupyter and Anaconda, as they're Python things ("things" is the technical jargon, btw).

**Comment by aspencoyle** [comment, marked as answer]
I've got a few ideas on what the issue might be!

First, gene_trans_map requires a string - try inputting `'none'` instead of `none`.

If that doesn't fix it, the issue might be with out_prefix - try changing it to `--out_prefix kallisto \` and see if output files show up in the directory you're running the script from

Finally, when I ran it I kept getting those long complex error messages (even after getting output that looked fine) and Steven basically told me "all good, don't worry about them". I assume that's probably true for you too?

**Reply by meganewing** [reply]
That worked! Thank you :)
